RELEASE.md

```diff
@@ -35,7 +35,7 @@
 * Bumped the upper bound for the Flake8 dependency to <5.0.
 * `kedro jupyter notebook/lab` no longer reuses a Jupyter kernel.
 * Required `cookiecutter>=2.1.1` to address a [known command injection vulnerability](https://security.snyk.io/vuln/SNYK-PYTHON-COOKIECUTTER-2414281).
-
+* Reduced constructor arguments for `APIDataSet`, and made it consistent w.r.t. requests API, automatically enabling the full configuration domain: stream, certificates, proxies, and more.
```

Comment:
Since this is going into `develop` it won't be released until 0.19. Hence the release docs belong in the "Upcoming Release 0.19.0" section above instead.

I'd put this in a new `### DataSets` section under "Breaking changes to the API" and also "Migration guide from Kedro 0.18.* to 0.19.*" that shows how someone should alter their existing `data`/`header` etc. arguments to sit under `load_args`. See https://github.com/kedro-org/kedro/blob/main/RELEASE.md#release-0180 for an example of how we've done this before.

Reply:
Yep, thanks for pointing this out!